faucet-server/Sources/App/Extensions/EnvironmentExtensions.swift

```diff
@@ -9,18 +9,33 @@ import Vapor
 
 extension Environment {
     struct Process {
+        // ckb
         static var nodeURL: String!
-        static var oauthClientId: String!
-        static var oauthClientSecret: String!
         static var walletPrivateKey: String!
         static var sendCapacityCount: Decimal!
+        // oauth
+        static var oauthClientId: String!
+        static var oauthClientSecret: String!
+        // db
+        static var dbHostname: String!
```

Comment:
As configuration becomes complicated, perhaps it's time we grouped related properties into sub structures. For example, putting db properties inside `Environment.Process.db`.

Reply:
https://github.com/nervosnetwork/ckb-testnet-faucet/pull/85/commits/42e1e0ab8146791510e2a07b9b01843320484d3a